src/sentry/templatetags/sentry_react.py

```diff
@@ -70,22 +70,16 @@ def _get_statuspage():
 @register.simple_tag(takes_context=True)
 def get_react_config(context):
     if 'request' in context:
-        user = getattr(context['request'], 'user', None) or AnonymousUser()
-        messages = get_messages(context['request'])
-        session = getattr(context['request'], 'session', None)
-        try:
-            is_superuser = context['request'].is_superuser()
-        except AttributeError:
-            is_superuser = False
+        request = context['request']
+        user = getattr(request, 'user', None) or AnonymousUser()
+        messages = get_messages(request)
+        session = getattr(request, 'session', None)
+        is_superuser = is_active_superuser(request)
     else:
         user = None
         messages = []
         is_superuser = False
 
-    if user:
-        user = extract_lazy_object(user)
-        is_superuser = user.is_superuser
```

Comment:
This would've been overriding the `is_active_superuser` value. We probably don't want this at all, but correct me if I'm wrong.